src/main/kotlin/uk/gov/justice/digital/hmpps/approvedpremisesapi/service/EmailNotificationService.kt

```diff
@@ -19,8 +19,13 @@ class EmailNotificationService(
 ) : EmailNotifier {
   var log: Logger = LoggerFactory.getLogger(this::class.java)
 
-  override fun sendEmail(recipientEmailAddress: String, templateId: String, personalisation: Map<String, *>) {
-    applicationEventPublisher.publishEvent(SendEmailRequestedEvent(EmailRequest(recipientEmailAddress, templateId, personalisation)))
+  override fun sendEmail(
+    recipientEmailAddress: String,
+    templateId: String,
+    personalisation: Map<String, *>,
+    replyToEmailId: String?,
```

Comment:
As we're adding a new reply-to email address into Notify, does this break the default behaviour for CAS1? Does Notify know to use the first/oldest one as the default when we don't pass an ID?

If it's not clear, do we need to add the replyToID in explicitly for CAS1 emails to avoid breaking their reply to?

Reply:
it (hopefully) doesn't break the default behaviour for CA1, as the original email is set as the default one in Notify, and the default is used unless another is specified. We've also set the code here to make the param optional so we don't need to touch the other CAS methods that are sending emails using the default address - we've already added in the new email address on Notify and emails are working currently, so think it should be safe?